core/server/api/v2/utils/serializers/input/posts.js

```diff
@@ -39,6 +47,7 @@ module.exports = {
             }
             // CASE: the content api endpoint for posts should not return mobiledoc
             removeMobiledocFormat(frame);
+            includeTags(frame);
```

Comment:
I think we can put this behind `labs.isSet('members')` check, so the additional overhead of including tags in all posts request is minimized to when the members flag is enabled.